### PR TITLE
Add CEPH metadata file

### PR DESCRIPTION
**Pull request by paulocv**
This pull only adds our team-model metadata file. The first forecast submission will be included in another pull request.

### CODE REVIEW

File: model-metadata/CEPH-Rtrend_rsv.yaml
Comment (kjsato) [inline model-metadata/CEPH-Rtrend_rsv.yaml:1]: Thanks for your submission!

First, could you add the lines in your yaml below?

1. schema_version (please add this in the first line as is)
schema_version: "https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v2.0.
0/admin-schema.json"

2. model_version (required for hubValidation, so of course you can change it depending on your model's version, for example:)
model_version: "1.0"

**Review by kjsato — COMMENTED** [review]
Thanks for your submission!

First, could you add the lines in your yaml below?

schema_version (please add this in the first line as is)
schema_version: "https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v2.0.
0/admin-schema.json"

model_version (required for hubValidation, so of course you can change it depending on your model's version, for example:)
model_version: "1.0"

**Review by kjsato — COMMENTED** [review]
Thanks for your submission!

First, could you add the lines in your yaml below?

schema_version (please add this in the first line as is)
schema_version: "https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v2.0.
0/admin-schema.json"

model_version (required for hubValidation, so of course you can change it depending on your model's version, for example:)
model_version: "1.0"

**Comment by paulocv** [comment]
> Thanks for your submission!
> 
> First, could you add the lines in your yaml below?
> 
> schema_version (please add this in the first line as is) schema_version: "https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v2.0. 0/admin-schema.json"
> 
> model_version (required for hubValidation, so of course you can change it depending on your model's version, for example:) model_version: "1.0"

Thank you for the guidelines. I am now updating all the files on pull request #6 named `CEPH_2024-02-25`  – both the metadata and the forecast file. My latest update had a validation error that I could not understand from the logs. Thank you.

**Pull request closed without merging**